java/code/src/com/suse/manager/matcher/MatcherJsonIO.java

```diff
@@ -136,10 +136,10 @@ public MatcherJsonIO() {
                 from -> monitoringProductId.ifPresent(to -> lifecycleProductsTranslation.put(from, to)));
 
         selfProductsByArch = new HashMap<>();
-        // SUSE Manager Server 3.1 x86_64 & SUSE Linux Enterprise Server 12 SP2 x86_64
-        selfProductsByArch.put(AMD64_ARCH_STR, Arrays.asList(1518L, 1357L));
-        // SUSE Manager Server 3.1 s390 & SUSE Linux Enterprise Server 12 SP2 s390
-        selfProductsByArch.put(S390_ARCH_STR, Arrays.asList(1519L, 1356L)); // SUSE Manager Server 3.1 x86_64
+        // SUSE Manager Server 4.0 x86_64 & SUSE Linux Enterprise Server 15 SP1 x86_64
+        selfProductsByArch.put(AMD64_ARCH_STR, Arrays.asList(1899L, 1763L));
+        // SUSE Manager Server 4.0 x86_64 & SUSE Linux Enterprise Server 15 SP1 s390
+        selfProductsByArch.put(S390_ARCH_STR, Arrays.asList(1898L, 1762L));
```

Comment:
Hmm, I see multiple problems, but changing the IDs is makes sense in general. But:

- With 4.0 we are a base product. So we only need one (1) product id. SLES15 SP1 is not needed
- We miss ppc64le. This needs to be added as well.
- The comment for s390 is wrong - you say x86_64

Reply:
Good points. Addressed in the latest commits. Thanks!